### ISSUE TITLE
[BUG] Wrong encoding

### ISSUE THREAD

**Issue by luizhrios** [markdirish/node-odbc#242]
**Describe your system**
* `odbc` Package Version: latest
* ODBC Driver: SAP MaxDB
* Database Name: MaxDB
* Database Version:
* Database OS:
* Node.js Version: 14.15.1
* Node.js OS: Windows

**Describe the bug**
Bad characters on result, probably different encoding

For example letter Ç is broken

**What I Tried**
Changing connection string to include charset or CCSID to 1208 as it is said on other issues, no difference.

**Expected behavior**
Result to be as it is on the database, or a way to change the encoding.

**To Reproduce**
1. Connect to database
2. Create database table
3. Run query

**Code**
Not applicable, any SQL that has a content outside of ASCII table.

* The offending code:
Not applicable

**Additional context**
Using the same ODBC Driver on other software the result is normal

**Comment by luizhrios** [comment]
Apparently the same as 106

**Comment by markdirish** [comment]
Do you know how data is encoded in the database? Like the table/field encoding?

You mention `Using the same ODBC Driver on other software the result is normal`. Which software is that? In Node.js, I can only create string from UTF-8 or UTF-16.

> ODBC Driver: SAP MaxDB
> Database Name: MaxDB
> ...
> Changing connection string to include charset or CCSID to 1208 as it is said on other issues, no difference.

Those keywords are for different drivers, they won't have any effect on MaxDB. Looking at [the SAP MaxDB documentation for datasources](https://help.sap.com/docs/JAVA_SERVER_FOR_S4HANA/679253cae6ab4103a75c7109f1478abb/0ec04cc664344b57921e8b4e5fcc3486.html?version=1909.000), I don't see any way to specify the host encoding...

I[ do see that there is both an ASCII driver](https://help.sap.com/docs/JAVA_SERVER_FOR_S4HANA/679253cae6ab4103a75c7109f1478abb/479813d4b0a04c7d9dc591b9b60c4ef4.html?version=1909.000) and a UNICODE driver, but that usually just defines the APIs used. If you aren't using the UNICODE driver, it might be worth a shot.

**Comment by luizhrios** [comment]
Encoding for the probably the whole database is ASCII, I can't change that.  The software I talked about is DBeaver, I can use both JDBC and ODBC drivers with no wrong encoding.
Yeah, I understand MaxDB doesn't have any of that keywords, I thought it would be helpful to mention I tried.

> I can only create string from UTF-8 or UTF-16.

Well, that would explain it, but what can I do then to get right encoding?

**Comment by luizhrios** [comment]
> Do you know how data is encoded in the database? Like the table/field encoding?

Encoding for probably the whole database is ASCII, I can't change that.  The software I talked about is DBeaver, I can use both JDBC and ODBC drivers with no wrong encoding.

> Those keywords are for different drivers, they won't have any effect on MaxDB

Yeah, I understand MaxDB doesn't have any of that keywords, I thought it would be helpful to mention I tried.

> I can only create string from UTF-8 or UTF-16.

Well, that would explain it, but what can I do then to get right encoding?
